test/release/examples/benchmarks/shootout/regexdna-redux.perfkeys

```diff
@@ -1,4 +1,4 @@
 # file: regexdnaredux-release.dat
 real
 verify:agggtaa\[cgt\]|\[acg\]ttaccct 2178
-verify:50833411
+verify:50833409
```

Comment:
This looks suspect to me. Why would the output of this benchmark change from removing a deprecated function?

Reply:
I believe those numbers are indicating which line # to check for the verify message. With the deprecation messages removed, the output is all shifted up by two lines. I'll rerun this as a `-performance` test to make sure the change is correct.